libpcsxcore/new_dynarec/new_dynarec.c

```diff
@@ -35,6 +35,7 @@ static int sceBlock;
 #endif
 
 #include "new_dynarec_config.h"
+#include "../psxhle.h" //emulator interface
```

Comment:
If you're including this please remove `psxHLEt[]` from libpcsxcore/new_dynarec/emu_if.h